build/_k-standard-goals.shade

```diff
@@ -187,6 +189,18 @@ default Quiet='${ false }'
     }
   }
 
+#nuget-verify target='test' if='File.Exists(PACKAGELIST_JSON_FILENAME)' description='Verify if all the packages are generated properly'
+  exec program='cmd' commandline='/C dnu commands install --source ${DNX_TOOLS_FEED} NugetPackageVerifier --packages ${BASE_DIR}\commands\' if='ShouldVerifyNupkgs'
```

Comment:
@victorhurdugaci true, and maybe eventually it'll use a private DNX, but in the meantime we'll just update the tool if it breaks.